spring-cloud-gateway-sample/pom.xml

```diff
@@ -24,6 +24,12 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
+			<exclusions> <!-- exclude metrics to verify that gateway doesn't fall over when it is not available -->
```

Comment:
Instead of doing this to exclude it for tests you can use `@ClassPathExclusions` in the test
https://github.com/spring-cloud/spring-cloud-commons/blob/6e9745b7a431ac2673ab5ee182f32a7e57310e87/spring-cloud-commons/src/test/java/org/springframework/cloud/client/loadbalancer/LoadBalancerAutoConfigurationTests.java#L21

Reply:
@ryanjbaxter That won't work because it requires ModifiedClassPathRunner which doesn't extend SpringJUnit4ClassRunner. We would need a new ModifiedClassPathRunner that extends SpringJUnit4ClassRunner and not BlockJUnit4ClassRunner.

Reply:
We do it all the time with out that.